src/utils/managers/ClearanceManager.ts

```diff
@@ -57,8 +58,8 @@ export default class {
 		);
 	}
 
-	async getCommandOverride(guildID: Snowflake, name: string): Promise<Record<string, any> | undefined> {
-		const response: Record<string, any> = await sequelize.query(
+	async getCommandOverride(guildID: Snowflake, name: string): Promise<GuildCommandOverride> {
```

Comment:
Is this function still able to resolve to `undefined`?

Reply:
that should still be able to return `undefined` incase nothing is found